File: src/back/php/php/fibonacci.php
<?php
function fibonacci(int $n) : int
{
    if ($n < 2) {
        return 1;
    } else {
        return fibonacci($n - 2) + fibonacci($n - 1);
    }
}
$fib = fibonacci(40);
echo 'fib='.$fib."\n";

File: back/golang/src/fibonacci/fibonacci.go
package main

import "fmt"

func main() {
	fmt.Print("fib=", fibonacci(40), "\n")
}

func fibonacci(n int32) int32 {
	if n < 2 {
		return 1
	} else {
		return fibonacci(n-2) + fibonacci(n-1)
	}
}
